settings.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from wger.settings_global import *
+
+# Use 'DEBUG = True' to get more details for server errors
+DEBUG = True
+TEMPLATES[0]['OPTIONS']['debug'] = True
+
+ADMINS = (
+    ('Your name', 'your_email@example.com'),
+)
+MANAGERS = ADMINS
+
+DATABASES = {
+'default': {
+    'ENGINE': 'django.db.backends.postgresql_psycopg2',
+    'NAME': os.environ.get('NAME'),
+    'USER': os.environ.get('USER'),
+    'PASSWORD': os.environ.get('PASSWORD'),
+    'HOST': os.environ.get('HOST'),
+    'PORT': '5432',
+    }
+}
+
+# Make this unique, and don't share it with anybody.
+SECRET_KEY = 'qi8da5)fd@k^3!mr5l85^)f+#$w%9jq2c&4rg)-s=w3)!dwnd4'
+
+# Your reCaptcha keys
+RECAPTCHA_PUBLIC_KEY = ''
+RECAPTCHA_PRIVATE_KEY = ''
+NOCAPTCHA = True
+
+# The site's URL (e.g. http://www.my-local-gym.com or http://localhost:8000)
+# This is needed for uploaded files and images (exercise images, etc.) to be
+# properly served.
+# SITE_URL = 'http://localhost:8000'
```

Comment:
Remove commented out code unless it's necessary

Reply:
- Resolved but i only left comments which redirect what is required or what the fields do.